fecgov/fec-cms
# bug-Candidate profile page inconsistent financial data( from candidate list page)

## Summary (required)

- Resolves #3009 
_Include a summary of proposed changes._
This ticket just fix bug of candidate profile page inconsistent data from candidate list.

## Impacted areas of the application
List general components of the application that this PR will affect:
all candidates, candidate profile pages


## Screenshots
wrong data display (before):
<img width="800" alt="wrong1" src="https://user-images.githubusercontent.com/24395751/61925199-3c3ae480-af39-11e9-813c-4f7aa4e68a26.png">


correct data display (after):
<img width="904" alt="correc1" src="https://user-images.githubusercontent.com/24395751/61925212-4bba2d80-af39-11e9-8109-d8e8d98f8b39.png">



## How to test
1) checkout feature branch
2) point dev api
3) go to all candidates page 
http://127.0.0.1:8000/data/candidates/?has_raised_funds=true&is_active_candidate=true

ex1: search `H6IN05257`  (only one election year)
click candidate name.
compare with production to see difference.
https://www.fec.gov/data/candidate/H6IN05257/

ex2: H8GA06195 (special election)
do same as above

The separate ticket will create later for 
1)candidate profile pagefrom globe search candidate 
2)some presidential candidate profile page wrong data. 
next ticket: [https://github.com/fecgov/fec-cms/issues/3076](https://github.com/fecgov/fec-cms/issues/3076)
